### DISCUSSION TITLE
Feature Request: Dashboard

### DISCUSSION THREAD

**Discussion by rvigneshw** [edumeet/edumeet#850] [Ideas]
Wouldn't it be nicer to have a dashboard where all the stats are displayed like name of the rooms that are being used , CPU , Memory and bandwidth utilization of the server?

**Comment by christian-2** [comment]
The [Prometheus exporter](https://github.com/havfo/multiparty-meeting/issues/222) is a starting point in this direction. It already reports names of rooms as well as bandwidth utilization per participant. You could get CPU and memory utilization of the server e.g. form [`node-exporter`](https://github.com/prometheus/node_exporter).

**Comment by Alex-YddeR** [comment]
Can you please post the steps to configure the same

**Comment by christian-2** [comment]
@Alex-YddeR I have opened misi/mm-ansible#7. Consider [upvoting](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-conversations-on-github#reacting-to-ideas-in-comments) it to establish priority. Otherwise `prom.md` is your starting point.

**Comment by cap19** [comment]
@christian-2 prom.md gives good basic overview.

I too am interested in how to configure/integrate this with Grafana. I've upvoted. Thanks!
